services/py-api/src/service/participants_service.py

```diff
@@ -10,6 +11,7 @@
 from src.database.transaction_manager import TransactionManager
 from src.server.exception import DuplicateEmailError, DuplicateTeamNameError
 from src.server.schemas.request_schemas.schemas import ParticipantRequestBody
+from src.server.handlers.participants_handlers import ParticipantHandlers
```

Comment:
Because the service layer is called by the handlers layer, by importing the ParticipantHandler you would cause the error of circular import. Learn more about it here: https://www.geeksforgeeks.org/python-circular-imports/. Please remove this import, you should not need it if your code is designed correctly

Moreover, this is also the reason that the tests are currently not passing.

Reply:
Fixed in 1bec98abe406eea6382a5453ac98f3e900d3ddd4